roles/installer/tasks/59_power_off_cluster_servers.yml

```diff
@@ -19,12 +19,12 @@
         - "{{ groups.workers | default([]) }}"
 
     - name: Power off hosts via IPMI
-      ipmi_power:
+      community.general.ipmi_power:
```

Comment:
@reihl in case you haven't seen this, today @tonyskapunk modified this task to use `ansible.builtin.command` module instead of any ansible ipmi module because of some deprecated python packages, please see #314